content/applications/hr/recruitment/recruitment-flow.rst

```diff
@@ -0,0 +1,191 @@
+<recruitment/settings>` menu, the resumé appears on the applicant's card, on the right-hand side.
+
+Send interview
+--------------
+
+At any point in the hiring process, an *interview* can be sent to the applicant to obtain more
+information. These interviews are custom-made, and can be formatted in a variety of ways.
+
+The *Surveys* application is used to send these *interviews*, therefore the *Surveys* application
+must be installed to send an interview to an applicant.
+
+Odoo uses the term *interview*, but these can be thought of as questionnaires, surveys, tests,
+certifications, etc. Custom interviews can be formatted to suit each individual job positions
+needs. For more information on creating and editing interview forms, refer to the
+:doc:`../../hr/recruitment/new_job` documentation.
+
+.. example::
+   A job position for a computer programmer could have an interview in the form of a programming
+   quiz to determine the skill level of the applicant. A job position for a restaurant server could
+   have a questionnaire inquiring about the applicant's availability, if the desired applicant needs
+   to be available on weekend evenings.
+
+To send an interview to an applicant, click on an applicant's card to navigate to a detailed view of
+the applicant's card. At the top-left of the applicant's card, click the :guilabel:`Send Interview`
+button.
+
+If the applicant's card has an email address on file, a :guilabel:`Send an interview` pop-up window
+appears, with all the information populated.
+
+.. note::
+   In order to send an email to an applicant, there must be an email address on the applicant's
+   card. If an email address is not entered on the applicant's card, when :guilabel:`Send Interview`
+   is clicked, an :guilabel:`Edit:(Applicant)` pop-up window appears on top of the :guilabel:`Send
+   an interview` pop-up window. Enter the email address in the :guilabel:`email` field, then click
+   :guilabel:`Save & Close`. Once the applicant's information is saved, the :guilabel:`Edit:
+   (Applicant)` closes and the :guilabel:`Send an interview` pop-up window remains.
+
+Pre-configured email templates in Odoo sometimes use dynamic placeholders, which are automatically
+filled with specific data when the email is sent. For example, if a placeholder for the applicant's
+name is used, it will be replaced with the actual name of the applicant in the email. For more
+detailed information on email templates, refer to the :doc:`../../general/companies/email_template`
+documentation.
+
+Add any additional recipients for the survey if more people should receive the email. If an email is
+in the database as a contact, add that contact in the :guilabel:`Recipients` field. If an email
+should be sent to someone who is not in the database as a contact, and they should **not** be added
+as a contact, add their email in the :guilabel:`Additional emails` field.
```

Comment:
In this section, I like how you've already referenced other docs to cut down on the length :smile: 

This is entirely optional and a suggestion to further cut down on length:

Is this information and the specific instructions for these actions covered in the Schedule interviews doc? If so, to avoid repeating information, maybe we can just summarize it. 

For example, for instructions like "click here to do this", we could rewrite it as "when sending emails, additional recipients can be added" and skip the step-by-step details.

Reply:
I actually double checked, and the information is not the same. I also checked to see if there as a general 'this is ow you send emails' doc, but there isn't one of those either. The schedule interviews one is all about starting in the calendar, and all the various specifications for that- so a lot of the fields don't match/is different enough that I think this is OK to keep in here. I completely understand (and agree) with your suggestion to cut down on the 'overly detailed' information, but I honestly don't see it anywhere else that I can link it to, so I think I will keep it for now.